include/fuse_common.h

```diff
@@ -22,6 +22,7 @@
 
 #include "fuse_opt.h"
 #include "fuse_log.h"
+#include <stdbool.h>
```

Comment:
This is an installed header and it doesn't require this include to parse, the extra include should be in the c code where it's required rather than where it has visibility outside the project.